### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Ask the team a question
+labels: question
+assignees: 'RDIL'
```
Comment (RDIL) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
assignees: ''
```
your choice if you want to keep this or not

**Comment by fkorotkov** [comment]
Can't commit your suggestion :-(

Screen Shot 2019-03-18 at 2 21 22 PM

**Comment by RDIL** [comment]
Did it

**Comment by fkorotkov** [comment]
Great! Thank you! Didn't know you can have different templates. It will be very useful!

**Comment by RDIL** [comment]
Yeah, just make sure to keep the base template file so Github knows you have more then one.

**Pull request merged**